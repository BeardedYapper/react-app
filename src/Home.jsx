import React, { Component } from 'react';
import Box from '@mui/material/Box';
import Typography from '@mui/material/Typography';
import { Link } from 'react-router-dom';
import Button from '@mui/material/Button';

class Home extends Component {
  render() {
    return (
      <Box style={{marginTop: 'auto'}}>
      <div>
        <Typography variant="h3" style={{marginTop: '20px', marginBottom: '20px'}}>
          Home
        </Typography>
        <div style={{display: 'flex',alignItems: 'center', margin: '20px', justifyContent: 'center'}}>
            <Button style={{margin:'10px'}} variant="outlined" color="primary" component={Link} to="/coursers">Courses</Button>
            <Button style={{margin:'10px'}} variant="outlined" color="primary" component={Link} to="/careers">Careers</Button>
        </div>
      </div>
    </Box>
    );
  }
}

export default Home;
