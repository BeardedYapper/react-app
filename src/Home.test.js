import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from './Home';

test('Render Home page', () => {
  const { container } = render(
    <Router>
    <Home/>
    </Router>,
  );
  expect(container.firstChild).toMatchSnapshot();
});