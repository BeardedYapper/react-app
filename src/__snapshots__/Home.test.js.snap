// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home page 1`] = `
<div
  class="MuiBox-root css-0"
  style="margin-top: auto;"
>
  <div>
    <h3
      class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      style="margin-top: 20px; margin-bottom: 20px;"
    >
      Home
    </h3>
    <div
      style="display: flex; align-items: center; margin: 20px; justify-content: center;"
    >
      <a
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/coursers"
        style="margin: 10px;"
        tabindex="0"
      >
        Courses
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
      <a
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/careers"
        style="margin: 10px;"
        tabindex="0"
      >
        Careers
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
  </div>
</div>
`;
